flaky-tests/frameworks/cypress.md

```diff
@@ -75,5 +69,11 @@ would produce output that looks like this:
   </testsuite>
 </testsuites>
 ```
-
 The default attributes [can be configured](https://www.npmjs.com/package/mocha-junit-reporter) with the `reporterOptions` argument in the Cypress config.
+
+
```

Comment:
extra space intended?

Reply:
Hmm. I think that snuck into the template. I'll fix it for all of the frameworks at once in the next PR.